spec/lib/permitted_event_expected_results.yml

```diff
@@ -1,305 +0,0 @@
-##################################### MANAGER #########################
```

Comment:
Just curious why this is being removed?

Reply:
It's been replaced with Rebeccas testing of state in a more conventional way whcih is easier to debug: ```foi_permitted_events_spec.rb```